### ISSUE TITLE
Desafio 12 Parallax não roda em Mobile

### ISSUE THREAD

**Issue by GraziFrancisco** [gustavoguanabara/html-css#547]
Boa tarde professor tudo 10?

Primeiramente gostaria de agradecer pelos cursos e pelo ótimo trabalho que desenvolve pra nós aqui na web!
Fiz o desafio 12 e notei que em nenhum mobile funciona o efeito parallax, ou seja, testei no devtools no desktop e está ok, mas ao abrir o link via mobile não roda!

conforme o vídeo abaixo.

https://github.com/gustavoguanabara/html-css/assets/129116654/c3f14112-edc2-4581-b781-d66201e54f93

**Comment by wallacemt** [comment]
Olá, tudo bom? Tenta ver sé não foi algum erro no CSS, porque fiz aqui e tá funcionando. Ou talvez no HTML, ou também pode ser o navegador, testa em outros para ver.

**Comment by GraziFrancisco** [comment]
Oi estou bem e vc? Estranho, porque eu estou usando o link do próprio professor, e o arquivo dele tambem não funciona, nem no meu celular, nem no da minha amiga o efeito parallax. @wallacemt vc pode me dizer qual o celular vc testou e qual o sistema operacional? Obrigada.

**Comment by wallacemt** [comment]
@GraziFrancisco Certo, fiz o teste no meu celular que tem a versão MIUI Global 14.0.8 no navegador Chorne versão 120.0.6099.231. No PC testei no Win 10 com o chorne também. Tenta acessar o meu para ver =
https://wallacemt.github.io/progeto-site-cordel/index.html
se não funcionar então deve ter alguma coisa estranha kkkkk

**Comment by BenjamimCS** [comment]
Isso é um problema no IOS: na Issue #193 conversam sobre isso, recomendo dá uma olhada. Resumindo o Safari tinha uma compatibilidade parcial (ele também reconhecia a propriedade mas não surtia efeito algum) mas que desde então olhando no [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/background-attachment#browser_compatibility) agora parece ter suporte completo. Veja Issue, mas já veja se seu Safari ou Chrome estão atualizado.

**Comment by GraziFrancisco** [comment]
@BenjamimCS Provavelmente é uma incompatibilidade do IOS com CSS, porque nos dois ap que testei são IOS, vi no post que #193 que vários outros usuários estão com o mesmo problema que eu. Fico mais tranquila que não foi um erro meu rs.. Obrigada!

**Comment by BenjamimCS** [comment]
@GraziFrancisco 
Pois é. Só ressaltando que o professor já falou no curso que a base do Chrome não é o Chromium mas sim o Safari, inclusive todos os navegadores são assim. Coisas de Apple.

Com JS você pode dá um jeito nisso, tem um link lá que eu deixei que é uma das formas.

**Comment by flameastro** [comment]
@GraziFrancisco Olá, você já resolveu o seu problema ou ainda precisa de ajuda??
